sdl_android/src/main/java/com/smartdevicelink/api/screen/ScreenManager.java

```diff
@@ -124,9 +142,13 @@ public void setTextField4Type(MetadataType textField4Type) {
 		this.textField4Type = textField4Type;
 	}
 
-	/*public void setSoftButtonObjects(ArrayList<SoftButtonObject> softButtonObjects) {
-		this.softButtonObjects = softButtonObjects;
-	}*/
+	public void setSoftButtonObjects(@NonNull List<SoftButtonObject> softButtonObjects) {
+		softButtonManager.setSoftButtonObjects(softButtonObjects);
+	}
+
+	public void setSoftButtonObjects(@NonNull SoftButtonObject softButtonObject) {
```

Comment:
Should this be plural? Maybe `setSoftButtonObject`?

Reply:
I wasnt sure whether it was best to keep the same method name but accept a singular object or introduce a second name that accepts a singular object. i guess it can be up for debate. In other areas of the library, we keep the same name such as in `MetadataTags`